examples/micro-ros_addtwoints_service/micro-ros_addtwoints_service.ino

```diff
@@ -0,0 +1,73 @@
+#include <micro_ros_arduino.h>
+#include <example_interfaces/srv/add_two_ints.h>
+#include <stdio.h>
+#include <rcl/error_handling.h>
+#include <rclc/rclc.h>
+#include <rclc/executor.h>
+#include <std_msgs/msg/int64.h>
+
+rcl_init_options_t options;
+rcl_node_options_t node_ops;
+rcl_node_t node;
+rclc_support_t support;
+rcl_allocator_t allocator;
+rclc_executor_t executor;
+
+rcl_service_options_t service_options;
+rcl_service_t serv;
+const rosidl_service_type_support_t * service_type_support;
+rcl_wait_set_t wait_set;
+rmw_request_id_t req_id;
+example_interfaces__srv__AddTwoInts_Response res;
+example_interfaces__srv__AddTwoInts_Request req;
+
+const char * service_name = "/add_two_ints";
+
+void setup() {
+ delay(1000); 
```

Comment:
why is this necessary?

Reply:
Not really necessary but useful  on some platforms where USB Serial takes a bit to start